WeScan/Common/Quadrilateral.swift

```diff
@@ -161,7 +195,14 @@ extension Quadrilateral {
         
         return Quadrilateral(topLeft: topLeft, topRight: topRight, bottomRight: bottomRight, bottomLeft: bottomLeft)
     }
-    
+  
+    func perimeter() -> Double {
```

Comment:
This only works to calculate the perimeter of parallelograms (where the sides are equal).
We are dealing with a Quadrilateral here, which makes the calculation a bit more complex. 
Thankfully we already have the `distanceTo(point: CGPoint)` function on `CGPoint`.

The function woukd look something like that:
```
func perimeter() -> CGFloat {
        return topLeft.distanceTo(point: topRight) + topRight.distanceTo(point: bottomRight) + bottomRight.distanceTo(point: bottomLeft) + bottomLeft.distanceTo(point: topRight)
    }
```

Reply:
This has been implemented, and documentation has been added for the function.